.pre-commit-config.yaml

```diff
@@ -29,3 +31,14 @@ repos:
     hooks:
     -   id: isort
         args: ["--profile", "black"]
+-   repo: local
```

Comment:
I'm quite certain there is a pylint repo entry point, shouldn't it be used instead?

Reply:
Good point! I will look into it

Reply:
Pylint seems to reccomend that we use the local repo https://pylint.pycqa.org/en/latest/user_guide/pre-commit-integration.html